baselines/tasks/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "tasks",
+  "version": "0.1.0",
+  "description": "Tasks client for Node.js",
+  "repository": "googleapis/nodejs-tasks",
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "main": "build/src/index.js",
+  "files": [
+    "build/src",
+    "build/protos"
+  ],
+  "scripts": {
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "docs": "jsdoc -c .jsdoc.js",
+    "predocs-test": "npm run docs",
+    "docs-test": "linkinator docs",
+    "fix": "gts fix",
+    "lint": "gts check",
+    "prepare": "npm run compile-protos && npm run compile",
+    "system-test": "c8 mocha build/system-test",
+    "test": "c8 mocha build/test"
```

Comment:
should linting be required for `test` too?
```
"test": "npm run lint && c8 mocha build/test"
```

Reply:
Thanks for the suggestion! I think we have this in synthtool set up:
https://github.com/googleapis/nodejs-tasks/blob/master/synth.py#L56
also for other baseline test (nodejs-tasks API is only for testing this bug fix), we have `npm run lint` for circleCI tests: https://github.com/googleapis/gapic-generator-typescript/blob/master/.circleci/config.yml#L80 If there are not errors in `gts fix` we are good to go with `gts check` :)